benchs/bench_quantizer.py

```diff
@@ -106,6 +106,12 @@ def eval_quantizer(q, xq, xb, gt, xt, variants=None):
     print("===== PQ")
     eval_quantizer(pq, xq2, xb2, gt, xt2)
 
+if 'plsq' in todo:
+    print("===== Product LSQ")
+    plsq = faiss.ProductLocalSearchQuantizer(d, 2, M // 2, nbits)
```

Comment:
is 2 sub-quantizers a sweet spot for the ProductAddtiveQuantizer?

Reply:
No, this is just a simple demo configuration.